.github/workflows/cargo.yml

```diff
@@ -3,68 +3,59 @@ name: cargo
 on: [pull_request]
 
 jobs:
-  # Ensure all Rust code is properly formatted with `rustfmt`.
+  # Enforce coding style
   cargo-fmt:
     runs-on: ubuntu-latest
     container: quay.io/enarx/fedora-test
     steps:
-    - uses: actions/checkout@v2
-    
-    - name: rustfmt
-      run: cargo fmt -- --check
+      - uses: actions/checkout@v2
+      - name: Code is formatted correctly.
+        run: cargo make check-format
 
-  # Build the Enarx workspace.
-  cargo-build:
+  # Verify compatible licensing.
+  cargo-deny:
     runs-on: ubuntu-latest
     container: quay.io/enarx/fedora-test
     steps:
-    - uses: actions/checkout@v2
-    
-    # Ensure the project builds correctly.
-    - name: Build
-      run: cargo build
+      - uses: actions/checkout@v2
+      - name: Dependencies are license-compatible.
+        run: cargo make deny
 
-  # Test the Enarx workspace.
-  cargo-test:
+  # Scan database for reported vulnerabilities.
+  cargo-audit:
     runs-on: ubuntu-latest
     container: quay.io/enarx/fedora-test
     steps:
-    - uses: actions/checkout@v2
-    
-    # Run any specified Rust tests.
-    - name: Run tests
-      run: cargo test
+      - uses: actions/checkout@v2
+      - name: No unresolved reported vulnerabilities.
+        run: cargo make audit
 
-    # Run Rust tests with OpenSSL-specific features enabled for SEV.
-    - name: Run tests (SEV-specific)
-      run: cargo test --manifest-path sev/Cargo.toml --verbose --features=openssl
-
-  # Lint check with rust-clippy.
```

Comment:
@connorkuehl Is this test no longer needed?

Reply:
@mbestavros It's still there, the diff is misleading because when I re-wrote the tests with 'cargo-make' I put them in a different ordering from top-to-bottom.

Reply:
Er woops, I didn't read closely enough. Let me add this back in

Reply:
I don't see an SEV-specific test with OpenSSL features enabled in the new diff, though. I'm wondering if that tests is obsolete. If not, we should have a `make` task for that.

Reply:
You're quite right, my order of operations here was bad and my force push came after my lunch break. I've amended the PR to retain the 'openssl' feature enablement for sev.